azure/src/main/java/org/apache/iceberg/azure/AuthType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.azure;
+
+public enum AuthType {
```

Comment:
Does this need to be public?

Reply:
Yes, this needs to be public.
`AuthType` is accessed in `AzureProperties`, `AzureBlobClientFactory`, and `IntegrationTests`.

Making it package-private would leave `AzureProperties` unaffected since both of them reside in `org.apache.iceberg.azure` package.
However, `AzureBlobClientFactory`, and `IntegrationTests` would throw a compilation error since they reside in `org.apache.iceberg.azure.blob` package.